kilkari/src/main/java/org/motechproject/nms/kilkari/domain/SubscriptionPack.java

```diff
@@ -13,21 +14,26 @@
 public class SubscriptionPack {
 
     @Field
+    @Unique
     private String name;
```

Comment:
One creates the constraint in the database.  The other does validation in
the code prior to saving.  If we had to do only one I would go for @Column,
but I did both.